oap-server/server-configuration/configuration-zookeeper/src/main/java/org/apache/skywalking/oap/server/configuration/zookeeper/ZookeeperConfigWatcherRegister.java

```diff
@@ -46,9 +51,28 @@ public ZookeeperConfigWatcherRegister(ZookeeperServerSettings settings) throws E
     @Override
     public Optional<ConfigTable> readConfig(Set<String> keys) {
```

Comment:
Please add the comments about what are the structure of the single and group value in the Zookeeper.